src/material-experimental/mdc-card/card.scss

```diff
@@ -1 +1,169 @@
-// TODO: MDC core styles here.
+@import '@material/card/mixins';
+@import '../mdc-helpers/mdc-helpers';
+@import '../../cdk/a11y/a11y';
+
+// TODO(jelbourn): move header and title-group styles to their own files.
+
+// Size of the `mat-card-header` region custom to Angular Material.
+$mat-card-header-size: 40px !default;
+
+// Default padding for text content within a card.
+$mat-card-default-padding: 16px !default;
+
+// Include all MDC card styles except for color and typography.
+@include mdc-card-without-ripple($query: $mat-base-styles-query);
+
+// Add styles to the root card to have an outline in high-contrast mode.
+// TODO(jelbourn): file bug for MDC supporting high-contrast mode on `.mdc-card`.
+.mat-mdc-card {
+  @include cdk-high-contrast {
+    outline: solid 1px;
+  }
+}
+
+// Title text within a card. MDC does not have a pre-made title section for cards.
+// Maintained here for backwards compatibility with the previous generation MatCard.
+.mat-mdc-card-title {
+  // Custom elements default to `display: inline`. Reset to 'block'.
+  display: block;
+
+  // Apply default padding for a text content region. Omit any bottom padding because we assume
+  // this region will be followed by another region that includes top padding.
+  padding: $mat-card-default-padding $mat-card-default-padding 0;
+}
+
+// Subtitle text within a card. MDC does not have a pre-made subtitle section for cards.
+// Maintained here for backwards compatibility with the previous generation MatCard.
+.mat-mdc-card-subtitle {
+  // Custom elements default to `display: inline`. Reset to 'block'.
+  display: block;
+
+  // Apply default padding for a text content region. Omit any bottom padding because we assume
+  // this region will be followed by another region that includes top padding.
+  padding: $mat-card-default-padding $mat-card-default-padding 0;
+}
+
+// Header section at the top of a card. MDC does not have a pre-made header section for cards.
+// Maintained here for backwards compatibility with the previous generation MatCard.
+.mat-mdc-card-header {
+  // The primary purpose of the card header is to lay out the title, subtitle, and image in the
+  // correct order. The image will come first, followed by a single div that will contain (via
+  // content projection) both the title and the subtitle.
+  display: flex;
+  flex-direction: row;
+
+  // Apply default padding for the header region. Omit any bottom padding because we assume
+  // this region will be followed by another region that includes top padding.
+  padding: $mat-card-default-padding $mat-card-default-padding 0;
+
+  // When a subtitle is inside of a header, we want to move it up slightly to reduce the space with
+  // the title, and add a margin bottom to create space underneath the header.
+  .mat-mdc-card-subtitle {
+    margin-top: -8px;
+    margin-bottom: 16px;
```

Comment:
Just asking out of curiosity: can't we calculate this somehow through variables? (it feels a bit like magic numbers to me)

Reply:
Changed to use `$mat-card-default-padding`